pyscriptjs/src/plugins/python/py_markdown.py

```diff
@@ -1,9 +1,11 @@
+import html
 from textwrap import dedent
 
+from js import console
 from markdown import markdown
 from pyscript import Plugin, console
```

Comment:
Since we are importing console on line 4, should we remove it from here?

```suggestion
from pyscript import Plugin
```